tests/wpt-harness/expectations/WebCryptoAPI/sign_verify/hmac.https.any.js.json

```diff
@@ -0,0 +1,125 @@
+{
+  "setup": {
+    "status": "PASS"
+  },
+  "generate wrong key step: HMAC with SHA-1 signing with wrong algorithm name": {
+    "status": "FAIL"
+  },
+  "generate wrong key step: HMAC with SHA-256 signing with wrong algorithm name": {
+    "status": "FAIL"
+  },
+  "generate wrong key step: HMAC with SHA-384 signing with wrong algorithm name": {
+    "status": "FAIL"
+  },
+  "generate wrong key step: HMAC with SHA-512 signing with wrong algorithm name": {
+    "status": "FAIL"
+  },
+  "generate wrong key step: HMAC with SHA-1 verifying with wrong algorithm name": {
+    "status": "FAIL"
+  },
+  "generate wrong key step: HMAC with SHA-256 verifying with wrong algorithm name": {
+    "status": "FAIL"
+  },
+  "generate wrong key step: HMAC with SHA-384 verifying with wrong algorithm name": {
+    "status": "FAIL"
+  },
+  "generate wrong key step: HMAC with SHA-512 verifying with wrong algorithm name": {
+    "status": "FAIL"
+  },
```

Comment:
The tests which fail, are expected to fail for our runtime as we do not (yet?) support `globalThis.crypto.subtle.generateKey`